publisher/socials/twitter.py

```
@@ -0,0 +1,121 @@
+            t = re.sub(r'<br\s*/?>', '\n', t)
+        
+            t = t.strip()
+            if t:  # пропускаем пустые твиты
+                clean_tweets.append((t, image_src))
+            
+        return clean_tweets
+    
+    def post(self, post):
+        self.log_template = f"[{self.full_name}: {post.account.title}][{post.short}]" + " {0}"
+
+        apiNew = tweepy.Client(
```

Ruff (0.12.2)

Local variable `apiNew` is assigned to but never used

Remove assignment to unused variable `apiNew`

(F841)